index.html

```diff
@@ -137,65 +173,105 @@ <h2 class="cont-header">Professional Art Printing Data</h2>
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you
+                    like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+                <a href=""> <button class="resume">Get My Resume</button></a>
             </div>
-            <div class="skillscont">
-                <div class="lang-heading"><img src="images/circle.png" alt="">
-                    <h3 class="iconic"></h3>Skills</div>
-                <ul class="langlist">
-                    <li class ="jss">Codekit</li>
-                    <li class="jss">Github</li>
-                    <li class="jss">Codepen</li>
-                    <li class="jss"> Gitlab</li>
-                    <li class="jss"> Terminal</li>
-                </ul>
+
+            <div class="underline"></div>
+            <div class="lang-cont">
+                <div class="language-container">
+                    <div class="lang-heading"><img src="images/rhombus.png" alt="">
+                        <h3 class="iconic">Language</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Javascript</li>
+                        <li class="jss">Ruby</li>
+                        <li class="jss">Html</li>
+                        <li class="jss"> Css</li>
+                    </ul>
+                </div>
+                <div class="framecont">
+                    <div class="lang-heading"><img src="images/rectangle.png" alt="">
+                        <h3 class="iconic">Framework</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Bootstrap</li>
+                        <li class="jss">Ruby on Rails</li>
+                        <li class="jss">RSPec</li>
+                        <li class="jss">CapyBara</li>
+                        <li class="jss">Selenium</li>
+                    </ul>
+                </div>
+                <div class="skillscont">
+                    <div class="lang-heading"><img src="images/circle.png" alt="">
+                        <h3 class="iconic"></h3>Skills
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Codekit</li>
+                        <li class="jss">Github</li>
+                        <li class="jss">Codepen</li>
+                        <li class="jss"> Gitlab</li>
+                        <li class="jss"> Terminal</li>
+                    </ul>
+                </div>
             </div>
-        </div>
 
     </section>
 
+    <form action="https://formspree.io/f/mlevdeoa" method="post" class="contact-form">
+        <div class="interest">
+            <h5>I'm always interested in hearing about new projects,
+                so if you'd like to chat please get in touch.</h5>
```

Comment:
- [ ] Adjust the padding here
![image](https://user-images.githubusercontent.com/62705947/186166398-ca8f165f-8188-4471-83bb-a492b192105e.png)

- [ ] Also update the font-size/weight and line-height in order to match the design
![image](https://user-images.githubusercontent.com/62705947/186166696-2639ea88-600c-46e8-aeb8-ad3a6dfba01c.png)